cul.js

```diff
@@ -105,10 +105,15 @@ var Cul = function (options) {
     serialPort.on("open", function () {
 
         if (options.init) {
-            that.write(options.initCmd, function (err, res) {
+            that.write(options.initCmd, function (err) {
                 if (err) throw err;
+            });
+            serialPort.drain(function(err){
```

Comment:
why call drain here again? you already use it in the Cul.write method (Line 116) or am I misunderstanding this?